package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "slack-cloudflare-workers",
-  "version": "1.3.2",
+  "version": "1.3.8",
```

Comment:
I don't think this change is necessary

Reply:
I assumed you were trying to keep them parallel to each other, no?

Reply:
No, this project depends on a range of slack-edge "^1.3.7", which means 1.3.7 or newer ones in 1.3 minor series. We don't release slack-cloudflare-workers each time slack-edge core library's patch verison is released.